src/env/mpicc.bash.in

```diff
@@ -254,7 +254,16 @@ if [ "$linking" = yes ] ; then
         if [ "$static_mpi" = no ] ; then
           $Show $CC ${final_cppflags} $PROFILE_INCPATHS ${final_cflags} ${final_ldflags} "${allargs[@]}" -I$includedir -L$libdir $PROFILE_PRELIB $PROFILE_FOO $rpath_flags -l@MPILIBNAME@ @LPMPILIBNAME@ $PROFILE_POSTLIB ${final_libs}
         else
-          $Show $CC ${final_cppflags} $PROFILE_INCPATHS ${final_cflags} ${final_ldflags} "${allargs[@]}" -I$includedir -L$libdir $PROFILE_PRELIB $PROFILE_FOO $rpath_flags $libdir/libmpi.a @LPMPILIBNAME@ $PROFILE_POSTLIB ${final_libs}
+          fabric_dep=""
+          if [ "@ofi_embedded@" = yes ] ; then
+              fabric_dep=`pkg-config --static --libs $libdir/lib/pkgconfig/libfabric.pc`
```

Comment:
So close! This should just be `$libdir/pkgconfig/libfabric.pc`. Can you fix that in all files?

Reply:
🤦‍♂️ Yep...done.